src/components/PackageDashboard/PackageTable.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from 'react';
+import ReactTable from 'react-table';
+import ReactGA from 'react-ga';
+import PropTypes from 'prop-types';
+import Moment from 'moment';
+
+const PACKAGE_ID_LABEL = "Package ID";
+const PACKAGE_TYPE_LABEL = "Package Type";
+const SUBMITTER_LABEL = "Submitter";
+const TIS_NAME_LABEL = "TIS Name";
+const DATE_SUBMITTED_LABEL = "Date Submitted";
+
+const PACKAGE_ID = "packageId";
+const SUBMITTER_ID = "displayName";
+const PACKAGE_TYPE_ID = "packageType";
+const TIS_NAME_ID = "tisName";
+const DATE_SUBMITTED_ID = "createdAt";
+const DATE_FORMAT = "MM/DD YYYY, h:mm a";
```

Comment:
I think this is a different date format than we have been using in the apps.  Can you verify with Michael/Becky/Rachel that it is okay that we are presenting it differently here?

Reply:
Rachel is okay with the date format being different.